specification/repository.md

```diff
@@ -191,6 +191,22 @@ to the repo.
   - SHOULD define maximum supported version of each auto-instrumentation framework
 - MAY host the documentation in the [Observability Cloud documentation repository](https://github.com/splunk/public-o11y-docs)
 
+## Deployments
+
+For this section, "deployments" refers to software tools, utilities, and/or
+configuration for packaging, distributing, and/or installing `splunk-otel`
+GDI software. Typically, these are 3rd-party frameworks like ansible, puppet,
+chef, cloudfoundry, etc.
+The `splunk-otel-collector`
+[deployments](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments)
+are a good example of this.
+
+- Splunk O11y GDI repos MAY include one or more distributions.
```

Comment:
I get what you mean by `distributions` here but I'm not sure it is a commonly used work in this context.

Reply:
I meant deployments! Doh. Fixing....